Dockerfile-bot

```diff
@@ -1,6 +1,23 @@
 FROM docker.io/node:alpine
+
+RUN apk -U upgrade --no-cache
+
 WORKDIR /bot
+
 COPY package.json .
-RUN yarn install
-COPY . .
-CMD yarn bot
+COPY yarn.lock .
```

Comment:
> If you want to reduce docker image size you can add multistage builds here and copy only builded files to final image

Multi-stage container builds IMO add extra complexity and tech debt to worry about, the build process is quite simple here and we will never need anymore files than we have here.

Previously the container was copy over literally _ALL_ the files i.e

```
COPY . .
```

This can slow down pipelines significantly and bloats the container size